python/packages/autogen-agentchat/src/autogen_agentchat/teams/_group_chat/_events.py

```diff
@@ -7,8 +9,8 @@
 class GroupChatStart(BaseModel):
     """A request to start a group chat."""
 
-    message: ChatMessage | None = None
-    """An optional user message to start the group chat."""
+    message: ChatMessage | List[ChatMessage] | None = None
+    """An optional user message or list of messages to start the group chat."""
```

Comment:
How about rename the field to `messages`, and always use a list of messages?

Reply:
Renamed the field to messages and ensured to always use list of `ChatMessage`